changelog.d/11786.bugfix

```diff
@@ -0,0 +1 @@
+Fix a bug introduced in Synapse 1.46.0 that prevented `on_logged_out` module callbacks from being correctly awaited by Synapse.
```

Comment:
Did `on_logged_out` exist before that version?

Reply:
Not in the new module system, but in the legacy password auth provider interface yes